frontends/mit-learn/src/pages/HomePage/TestimonialsSection.tsx

```diff
@@ -296,15 +307,15 @@ const SlickCarousel = () => {
 const TestimonialsSection: React.FC = () => {
   return (
     <Section>
-      <Container id="hamster-noises">
+      <HeaderContainer id="hamster-noises">
```

Comment:
Hamsters make weird noises, so we probably don't want those noises on the website. Suggest removing this.

No offense to hamsters.